src/web/components/Core/InlineMessage.tsx

```diff
@@ -12,7 +12,7 @@ export type InlineMessageProps = {
 export function InlineMessage({ message, type }: InlineMessageProps) {
   return (
     <div className={clsx('inline-message message-container', type)}>
-      <span data-testid='banner-message' className='banner-text'>
+      <span data-testid='banner-message' className='banner-text' role='alert'>
```

Comment:
i don't know if you want to do this.  pretty sure this makes it always show up red.  we have a type prop for this.

Reply:
Removed